tools/build_defs/pkg/pkg.bzl

```diff
@@ -92,6 +92,8 @@ def _pkg_tar_impl(ctx):
         if dotPos > 0:
             dotPos += 1
             args += ["--compression=%s" % ctx.attr.extension[dotPos:]]
+        elif ctx.attr.extension == "tgz":
```

Comment:
This is right, but it is confusing. because the implementation is a little too clever. It happens to be that all the allowed file types are of the form .tar.XX where XX is a valid compression.
To make this more readable, ee should simply have a map of allowed file extensions to the compression method they belong with. But that is not your fault, so I won't block merge on that..